pkg/controller/multiarchbuildconfig/multiarchbuildconfig_test.go

```diff
@@ -25,41 +26,115 @@ import (
 	"github.com/openshift/ci-tools/pkg/manifestpusher"
 )
 
+var (
+	scheme *runtime.Scheme
+)
+
+type mockManifestPusher struct {
+	errToReturn error
+}
+
+func (m *mockManifestPusher) PushImageWithManifest(builds []buildv1.Build, targetImageRef string) error {
+	return m.errToReturn
+}
+
+type buildBuilder struct {
+	name     string
+	arch     string
+	phase    buildv1.BuildPhase
+	mabcName string
+}
+
+func NewBuildBuilder() *buildBuilder {
```

Comment:
Why do we need this, where you can just pass the build directly to the test case?

Reply:
Refactor build definition and save some lines. I can change if you prefer the build declaration to be inlined, I'm not opinionated here.

Reply:
Let's utilize this not only in the tests and keep it.